tests/lib/rules/no-multiple-empty-lines.js

```diff
@@ -82,7 +82,7 @@ ruleTester.run("no-multiple-empty-lines", rule, {
             options: ruleArgs
         },
         {
-            code: "// invalid 3\nvar a=5;\n\n\n\nvar b = 3;",
+            code: "// invalid 3\nvar a=5;\n\n\n\nvar b = 3;\n",
```

Comment:
This doesn't seem correct - you changed the code but the expected error is the same? It would be better to leave existing tests alone and add a new test that explicitly shows the problem.